GOVERNANCE.md

```diff
@@ -79,9 +79,9 @@ a standard TC motion.
 Changes to TC membership should be posted in the agenda, and may be
 suggested as any other agenda item (see "TC Meetings" below).
 
-No more than 1/3 of the TC members may be affiliated with the same
+No more than 1/4 of the TC members may be affiliated with the same
```

Comment:
What if the restriction was the lesser of 1/3 or 3(or 4?) people so that with a small TC of 6 you still have 2 people, but if it grows to a lot more you don't end up with massive over-representation from any one company.